torch/_higher_order_ops/invoke_subgraph.py

```diff
@@ -59,7 +59,59 @@ def __call__(
         return super().__call__(subgraph, identifier, operands)
 
 
+class InvokeSubgraphPlaceholderHOP(HigherOrderOperator):
```

Comment:
Does this need to be a HOP? For torch.utils.checkpoint, we transformed the torch.utils.checkpoint function into the checkpoint HOP. So we can probably get away with this just being an `invoke_subgraph_placeholder` function